package.json

```diff
@@ -137,7 +137,6 @@
     "querystring": "^0.2.1",
     "ramda": "^0.28.0",
     "redis": "^4.5.1",
-    "request": "^2.88.2",
```

Comment:
`npm audit fix` reported [vulnerabilities with GHSA-72xf-g2v4-qvf3|request>tough-cookie](https://app.circleci.com/pipelines/github/ministryofjustice/licences/2974/workflows/3bc0608f-bb98-4935-8124-f1825ff01d1e/jobs/20206) which did not have a fix. request doesn't seem to be used in the rest of the service so removed this from `package.json`